solutions/gold/cf-585D.mdx

```diff
@@ -89,7 +89,7 @@ int main() {
 	vector<array<int, 4>> right = calc_subarray(n / 2, n);
 	sort(begin(right), end(right));
 
-	const int INF = 1e9;
+	const int INF = INT32_MAX;
```

Comment:
i mean now there's no need for inf, right

Reply:
ok that is true, ima change that in a sec